packages/base/any/kernels/5.8-lts/patches/0014-delta-tx4810-dn-dts.patch

```diff
@@ -0,0 +1,171 @@
+--- a/arch/arm64/boot/dts/marvell/delta-tx4810-dn.dts
```

Comment:
Please add a git commit message, and also sign it of. Is this in Linux upstream already?

Reply:
This dts is not in the Linux upstream, so I changed the source to /dev/null, and hope it is more clear now.